[wasm] Wasm.Build.Tests build failing with `Workload ID wasm-tools is not recognized`

Wasm.Build.Tests CI builds are [failing](https://dev.azure.com/dnceng/public/_build/results?buildId=1920824&view=logs&jobId=6f051c1a-c3e7-5cde-411e-64125624a208&j=6f051c1a-c3e7-5cde-411e-64125624a208&t=b731ef43-7f67-5a47-0aca-433fa8d3212b) with:

```
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Workload ID wasm-tools is not recognized. [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
##[error]eng/testing/workloads-testing.targets(218,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) Workload ID wasm-tools is not recognized.
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Workload ID wasm-tools is not recognized. [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
##[error]eng/testing/workloads-testing.targets(218,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) Workload ID wasm-tools is not recognized.
  Exit code: 1
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : workload install failed with exit code 1: Welcome to .NET 7.0! [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : --------------------- [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : SDK Version: 7.0.100-rc.1.22403.4 [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : ---------------- [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Installed an ASP.NET Core HTTPS development certificate. [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : To trust the certificate run 'dotnet dev-certs https --trust' (Windows and macOS only). [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Learn about HTTPS: https://aka.ms/dotnet-https [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : ---------------- [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Write your first app: https://aka.ms/dotnet-hello-world [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Find out what's new: https://aka.ms/dotnet-whats-new [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Explore documentation: https://aka.ms/dotnet-docs [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Report issues and find source on GitHub: https://github.com/dotnet/core [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : -------------------------------------------------------------------------------------- [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
/__w/1/s/eng/testing/workloads-testing.targets(218,5): error : Workload ID wasm-tools is not recognized. [/__w/1/s/src/tests/BuildWasmApps/Wasm.Build.Tests/Wasm.Build.Tests.csproj]
##[error]eng/testing/workloads-testing.targets(218,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) workload install failed with exit code 1: Welcome to .NET 7.0!
```

This is with the latest sdk - `7.0.100-rc.1.22403.4 `.
Sdk side issue: https://github.com/dotnet/sdk/issues/26967

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -163,7 +163,7 @@
     <MoqVersion>4.12.0</MoqVersion>
     <FsCheckVersion>2.14.3</FsCheckVersion>
     <!-- Uncomment to set a fixed version, else the latest is used -->
-    <SdkVersionForWorkloadTesting>7.0.100-rc.1.22402.35</SdkVersionForWorkloadTesting>
+    <!--<SdkVersionForWorkloadTesting>7.0.100-rc.1.22402.35</SdkVersionForWorkloadTesting>-->
     <CompilerPlatformTestingVersion>1.1.2-beta1.22205.2</CompilerPlatformTestingVersion>
     <!-- Docs -->
     <MicrosoftPrivateIntellisenseVersion>7.0.0-preview-20220721.1</MicrosoftPrivateIntellisenseVersion>

